functions/helloworld/hello-pubsub/src/test/java/functions/ExampleIT.java

```diff
@@ -60,13 +65,21 @@ public static void setUp() throws IOException {
 
     // Emulate the function locally by running the Functions Framework Maven plugin
     emulatorProcess = new ProcessBuilder()
-        .command("mvn", "function:run")
+        .command("/usr/local/bin/mvn", "function:run")
```

Comment:
```suggestion
        .command("mvn", "function:run")
```

Reply:
Good catch - meant to revert this before committing. 😢